tdrs-frontend/src/actions/reports.js

```diff
@@ -183,10 +184,17 @@ export const submit =
         })
         removeFileInputErrorState()
 
-        const submittedFiles = responses.map(
-          (response) =>
-            `${response?.data?.original_filename} (${response?.data?.extension})`
-        )
+        const submittedFiles = responses.reduce((result, response) => {
+          const submittedFile = response?.data
+          dispatch({
```

Comment:
Each file needs to be marked as submitted individually, to increase performance `reduce` was used to allow one iteration through the responses.